web/styles/popovers.css

```diff
@@ -363,7 +363,8 @@ ul {
     background-position: center;
     position: relative;
 
-    &.guest-avatar::after {
+    &.guest-avatar::after,
+    &.bot-avatar::after {
```

Comment:
This outline is added for `guest-avatar` in other places as well, like in user profile modal and avatar shown with message. So we might also add it to `bot-avatar` in other places as well. We can actually have it in the common CSS itself in `app_components.css` by having a prep commit to move this there.

Reply:
Done.
Also, we reduce the size of the outline since it covered most part of the avatar.